src/components/comment-input.component.js

```diff
@@ -113,12 +120,12 @@ export class CommentInput extends Component {
                   : translate('issue.main.commentInput', language)
               }
               multiline
-              blurOnSubmit
+              blurOnSubmit={false}
               onChangeText={text => this.setState({ text })}
               onContentSizeChange={event =>
                 this.setState({ height: event.nativeEvent.contentSize.height })}
               onSubmitEditing={event =>
```

Comment:
Yes, but even if it is removed (or `false`), the problem remains. As I wrote, this is due to non-standard keyboards (on Android at least). Therefore, need to add an new line. I do not know how things are on iOS? Maybe there is enough to remove `blurOnSubmit`.